srcpkgs/librespot/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'librespot'
+pkgname=librespot
+version=0.4.2
+revision=1
+build_style=cargo
+configure_args="--features alsa-backend,pulseaudio-backend"
+hostmakedepends="pkg-config"
+makedepends="alsa-lib-devel avahi-compat-libs-devel glib-devel libpulseaudio"
+depends="alsa-lib avahi-compat-libs glib libpulseaudio"
+short_desc="Open Source Spotify client library"
```

Comment:
is it a library or a program? the package as-is installs just a binary, and rust libraries aren't very useful as packages in most cases

Reply:
It is a program which runs a Spotify client where frontends like spotify-qt or even the official Spotify app can play music to. I believe it is a fork of spotifyd, which does the exact same thing.

Reply:
If you're talking about the description, that is how librespot is self described on github. Do you think its better to change it to prevent confusion?